Game/Source/GDKTestGyms/Private/NFRConstants.cpp

```diff
@@ -86,6 +86,17 @@ void UNFRConstants::InitWithWorld(const UWorld* World)
 		}
 	}
 	UE_LOG(LogNFRConstants, Log, TEXT("Min client FPS: %.8f."), MinClientFPS);
+
+	// Player Average Velocity
+	{
+		float OverrideMinPlayerAvgVelocity;
+		if (FParse::Value(FCommandLine::Get(), TEXT("MinPlayerAvgVelocity="), OverrideMinPlayerAvgVelocity))
```

Comment:
What is the convention for CommandLine arguments.. -MinPlayerAvgVelocity ?? also are these defined as consts somewhere?

Reply:
This argument is designed for the `native scenarios`. It override the average velocity, if the average velocity in test is below this given value, it will failed the test by adding `NFRFailureString`.

These arguments are defined in files like `ci\config\server_args\max_player_density.json`.

This only works for the `native scenarios`.
The `nfr scenarios` check the value through the query to the metrics.